include/transfer.h

```diff
@@ -307,7 +311,34 @@ namespace quda {
   void Restrict(ColorSpinorField &out, const ColorSpinorField &in, const ColorSpinorField &v, 
 		int Nvec, const int *fine_to_coarse, const int *coarse_to_fine, const int * const *spin_map,
 		int parity=QUDA_INVALID_PARITY);
-  
+
+
+#ifdef GPU_STAGGERED_DIRAC
```

Comment:
We prefer to define all functions in headers without macro guards, and leave the macro enablement to the implementation, with an appropriate error message if a function is called but not compiled.

Reply:
Noted and fixed. `staggered_prolong_restrict.cu` already had the appropriate macro guards in it, so there were no changes needed there.